mantidimaging/core/imopr/visualiser.py

```diff
@@ -2,24 +2,29 @@
 
 from logging import getLogger
 
+import matplotlib.pyplot as plt
+from matplotlib.widgets import Slider
+from matplotlib.widgets import RectangleSelector
+from matplotlib.patches import Rectangle
+
+from mantidimaging.core.imopr.utility import handle_indices
+
 
 def sanity_checks(config):
     pass
 
 
 def execute(sample, flat, dark, config, indices):
-    from mantidimaging.core.imopr import helper
     getLogger(__name__).info("Running IMOPR with action VISUALISE/SHOW")
-    import matplotlib.pyplot as plt
```

Comment:
Is it possible this was intentionally imported only within the execution step as it's part of the `core`?